src/main/java/roomescape/web/RoomescapeApplication.java

```diff
@@ -1,9 +1,11 @@
-package roomescape;
+package roomescape.web;
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.ComponentScan;
 
 @SpringBootApplication
+@ComponentScan(basePackages = {"roomescape.web", "roomescape.core"})
```

Comment:
이렇게 한번에 선언할 수 있을것같아요!

```suggestion
@SpringBootApplication(scanBasePackages = {"roomescape.web", "roomescape.core"})
```